### PR TITLE
i18n(fr): Add `recipes/i18n` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `recipes/i18n` in French language

#### Related issues & labels (optional)

- Suggested label: I18n

### CODE REVIEW

**Review by McFlyPartages — COMMENTED** [review]
Goof job @thomasbnt .

Here are some suggestions

File: src/content/docs/fr/recipes/i18n.mdx [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+    ```
+
+## Ressources
+- [Choisir une étiquette de langue](https://www.w3.org/International/questions/qa-choosing-language-tags)
+- [Style de droite à gauche 101](https://rtlstyling.com/)
+
+## Bibliothèques communautaires
+- [astro-i18next](https://github.com/yassinedoghri/astro-i18next) — Une intégration d'Astro pour i18next comprenant quelques composants utilitaires.
+- [astro-i18n](https://github.com/alexandre-fernandez/astro-i18n) — Une bibliothèque d'internationalisation en premier lieu TypeScript pour Astro.
+- [astro-i18n-aut](https://github.com/jlarmstrongiv/astro-i18n-aut) — Une intégration Astro pour i18n qui supporte le `defaultLocale` sans génération de page. L'intégration est indépendante de l'adaptateur et du cadre d'interface utilisateur.
```
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/recipes/i18n.mdx:509]: Oui good!

**Review by McFlyPartages — APPROVED** [review]
LGTM

File: src/content/docs/fr/recipes/i18n.mdx [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+import ReadMore from '~/components/ReadMore.astro'
+import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
+
+Dans cette recette, vous apprendrez à utiliser les collections de contenus et le routage dynamique pour construire votre propre solution d'internationalisation (i18n) et servir votre contenu dans différentes langues.
+
+:::tip
+Dans la version 4.0, Astro a ajouté un support intégré pour le routage i18n permettant de configurer les langues par défaut et prises en charge, mais aussi, inclut des fonctions d'aide précieuses pour vous aider à servir un public international. Si vous souhaitez l'utiliser à la place, consultez notre [guide d'internationalisation](/fr/guides/internationalisation/) pour en savoir plus sur ces fonctionnalités.
```
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/recipes/i18n.mdx:14]: Oh good catch!

**Comment by thomasbnt** [comment]
> Fix link

Fixed! 🕺🏼

**Review by McFlyPartages — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🚀

**Pull request merged**